src/mlpack/tests/hmm_test.cpp

```diff
@@ -204,10 +204,10 @@ BOOST_AUTO_TEST_CASE(SimpleBaumWelchDiscreteHMM)
   observations.push_back("1 1 1 0 0 0 1 1 1 0 0 0");
   observations.push_back("0 1 0 1 0 1 0 1 0 1 0 1");
   observations.push_back("0 0 0 0 0 0 1 1 1 1 1 1");
-  observations.push_back("1 1 1 1 1 1 0 0 0 0 0 0");
-  observations.push_back("1 1 1 0 0 0 1 1 1 0 0 0");
-  observations.push_back("0 0 1 1 0 0 0 0 1 1 1 1");
-  observations.push_back("1 1 1 0 0 0 1 1 1 0 0 0");
+  observations.push_back("1 1 1 1 1 0 1 0 0 0 0 0");
+  observations.push_back("1 1 1 0 0 1 0 1 1 0 0 0");
+  observations.push_back("0 0 1 1 0 0 0 1 0 1 1 1");
+  observations.push_back("1 1 1 0 0 1 0 1 1 0 0 0");
```

Comment:
This was for the static code analyzer, which didn't like the duplicated observations.  But changing the order of observations doesn't affect the result.

Reply:
interesting, looks like after the update the static code analyzer is becoming picky